src/TestSuite/TestCase.php

```diff
@@ -203,13 +203,27 @@ public function withErrorReporting(int $errorLevel, callable $callable): void
      */
     public function deprecated(callable $callable): void
     {
-        $errorLevel = error_reporting();
-        error_reporting($errorLevel & ~E_USER_DEPRECATED);
+        $deprecation = false;
```

Comment:
```suggestion
        /** @var boolean $deprecation */
        $deprecation = false;
```

This might help with the error on 226